test/unit/cmap/connect.test.ts

```diff
@@ -224,13 +230,21 @@ describe('Connect Tests', function () {
 
         context('when 512 byte size limit is exceeded', () => {
           it(`should not 'env' property in client`, async () => {
-            // make metadata = 507 bytes, so it takes up entire LimitedSizeDocument
+            // make a metadata object that, with just the name and appName, is already at capacity.
             const longAppName = 's'.repeat(493);
+            const metadata = makeClientMetadata(
```

Comment:
The changes like this one in this file are a result of making `addContainerMetadata` internal to the client metadata module.

`makeClientMetadata` truncates the appname, so creating a long appName isn't sufficient to make the metadata exceed the handshake's max size anymore.